upcloud/resource_upcloud_objectstorage_test.go

```diff
@@ -15,27 +15,24 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
 	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
 	"github.com/minio/minio-go/v7"
+	"github.com/rs/xid"
 )
 
 const expectedDescription = "My object storage"
 const expectedZone = "pl-waw1"
 const expectedKey = "an access key"
 const expectedSecret = "a secret key"
+const runPrefix = "testacc-"
 
-const expectedName1 = "test-name1"
-const expectedName2 = "test-name2"
-const expectedName3 = "test-name3"
+var runID = xid.New().String()
+var expectedName1 = fmt.Sprintf("%s%s-1", runPrefix, runID)
+var expectedName2 = fmt.Sprintf("%s%s-2", runPrefix, runID)
+var expectedName3 = fmt.Sprintf("%s%s-3", runPrefix, runID)
 
 func init() {
```

Comment:
This is not part of changes but isn't this `init()` run every time our provider runs because our tests are in `upcloud` package ?  If so maybe we could rename it to eg `initObjectStorageTests()` and call it on each test function ?

Reply:
Good question. It'll run only when the `go test` is being run, and after all other `init()` calls have finished on non-test files. The Go compiler will ignore any `*_test.go` files from all packages so this piece of code won't affect the runtime.